tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1231,14 +1231,18 @@ a dictionary can also be used with a named enum.
 
     # Is the same as:
     const State = {STATE_IDLE = 0, STATE_JUMP = 5, STATE_SHOOT = 6}
+    # Access values with State.STATE_IDLE, etc.
 
     func _ready():
         # Access values with Name.KEY, prints '5'
         print(State.STATE_JUMP)
         # Use constant dictionary functions
         # prints '["STATE_IDLE", "STATE_JUMP", "STATE_SHOOT"]'
         print(State.keys())
-
```

Comment:
Should I restore the whitespace here? If yes, should I place it between the below print statements so they aren't a united codeblock?

Reply:
Put it after the last line of code, keep two lines before "Functions", easier to read